client/my-sites/checkout/src/components/wp-order-review-line-items.tsx

```diff
@@ -252,7 +252,7 @@ function LineItemWrapper( {
 	const shouldShowVariantSelector =
 		onChangeSelection &&
 		! isWooMobile &&
-		! isRenewal &&
+		( ! isRenewal || ( isRenewal && product.is_domain_registration ) ) &&
```

Comment:
Minor, but the mix of negatives, OR, and AND make this a bit hard to understand. We could make this more readable by creating another variable. Something like:

```js
const isNonDomainRenewal = isRenewal && ! product.is_domain_registration;
const shouldShowVariantSelector =
  // ...
  ! isNonDomainRenewal &&
  // ...
```

Reply:
Thanks, updated.